### ISSUE TITLE
[3.0.10] createReadStream does not return the instance of Readable.

### ISSUE THREAD

**Issue by mysticatea** [isaacs/node-graceful-fs#81]
I'm sorry for opening the issue of an old version.

7 days ago, `graceful-fs@3.0.10` was released.
I found the problem of a breaking change in `graceful-fs@3.0.10`.
`fs.createReadStream()` does not return the instance of `stream.Readable`.

```js
var fs = require("graceful-fs")
var Readable = require("stream").Readable

console.log(fs.createReadStream("test.js") instanceof Readable)
```

- `graceful-fs@3.0.9` ⇒ `true`
- `graceful-fs@3.0.10` ⇒ `false`

**Comment by isaacs** [comment]
Indeed!  That's because it's shimming *all* the internal modules, instead of just `fs`.

I think a simple fix is to make the `natives` module take a list of modules to not shim, and then graceful-fs can opt out of shimming the stream modules.

**Comment by isaacs** [comment]
Should be fixed on 3.0.11

**Comment by mysticatea** [comment]
I confirmed this issue has been fixed.
Thank you very much for the great support!
